Bearded.Monads.Tests/TryTests.cs

```diff
@@ -322,5 +334,24 @@ public void Associativity()
             Assert.Equal(second, first);
         }
         #endregion
+
+        #region Supporting methods
+        private static int ExecutionCount = 0;
```

Comment:
This is not really a good idea - if any subsequent tests refer to this, they will need to reset it - it would be much cleaner to just put this in the test as a local. It's also not threadsafe, so one can't run tests in parallel when they refer to this.

You can probably just pop the entire region into the body of the test and it'll work fine.

Reply:
You are right. Will do it :)